### PR TITLE
[DGPB-1930] index with suggester options

### PR DESCRIPTION
**Pull request by alfonsostratio**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
         <akka.version>2.5.19</akka.version>
         <debug.suspend>n</debug.suspend>
         <dg.commons.version>0.10.0-SNAPSHOT</dg.commons.version>
-        <dg.business.glossary>0.3.0-SNAPSHOT</dg.business.glossary>
+        <dg.business.glossary>0.3.0-M2</dg.business.glossary>
```
Comment (alfonsostratio) [inline pom.xml]: Temporal, fails with SNAPSHOT

File: dgindexer/src/test/scala/com/stratio/governance/agent/searcher/actors/manager/DGManagerJsonUnitTest.scala [hunk 2]
```diff
@@ -101,9 +101,9 @@ class DGManagerJsonUnitTest extends FlatSpec {
...
-  }
+  //"Model Json File " should " match to key list retrieved" in {
```
Comment (alfonsostratio) [inline dgindexer/src/test/scala/com/stratio/governance/agent/searcher/actors/manager/DGManagerJsonUnitTest.scala]: Uncomment and fix test

File: dgindexer/src/main/resources/se-manager/governance_domain_template.json [hunk 3]
```diff
@@ -3,6 +3,9 @@
   "model": {
     "id": "id",
     "language": "english",
+    "search_fuzzy_enabled": true,
```
Comment (alfonsostratio) [inline dgindexer/src/main/resources/se-manager/governance_domain_template.json]: TODO: Add     "search_phonetic_enabled": true,

**Comment by jbernal-stratio** [comment]
I close this PR and open other one (#45) with this changes and the unit and IT test, doc and changelog updated

**Pull request closed without merging**